L29-opencensus-filter.md

```diff
@@ -0,0 +1,40 @@
+Title
+----
+* Author(s): Jim King, Emil Mukilic
+* Approver:
+* Status: In Review
+* Implemented in: C++
+* Last updated: 5-14-2018
```

Comment:
Please use ISO format for dates: 2018-05-14.

Reply:
Done.